src/pages/confirmAppointment/[appointment_id].tsx

```diff
@@ -53,74 +110,165 @@ export default function Home() {
+
+              {/* Tags */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <Tags
+                  header="Tag"
+                  value={appointment.detailHeader.tags.map((t, index): Tag => {
+                    return { name: t, id: index };
+                  })}
+                  handleValueChange={() => {}}
+                  menuValue={[]}
+                  isErr={false}
+                  errMsg=""
+                  readOnly={true}
+                />
+                <FormHelperText>{"\u00A0"}</FormHelperText>
+              </Box>
+
+              {/* Description */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <DescriptionTextField
+                  name="description"
+                  header="Description"
+                  placeholder="Enter Description Here"
+                  value={appointment.detailHeader.description}
+                  handleValueChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  height={8}
+                  readOnly={true}
+                />
+              </Box>
+            </Stack>
+          </Card>
+          <Card sx={ConfirmAppointmentStyle.Card}>
+            <Stack spacing={3} alignItems="center" justifyContent="center">
+              {/* Image list */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <DisplayImages header="Image" images={appointment.images} />
+              </Box>
+
+              {/* Number of participants */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <Typography variant="h3">{`Number of Participant : ${appointment.acceptParticipants.length}`}</Typography>
+              </Box>
+
+              {/* Participant List */}
+              <Stack
+                spacing={0.5}
+                alignItems="start"
+                justifyContent="center"
+                sx={ConfirmAppointmentStyle.TextField}
+              >
+                <Typography variant="h2">Join with</Typography>
+                <Box display="flex">
+                  {appointment.acceptParticipants.length == 0 && (
+                    <Typography variant="body1" color="error">
+                      No one is interested in this activity yet.
+                    </Typography>
+                  )}
+                </Box>
+                <Grid container spacing={1} style={{ marginLeft: -5 }}>
+                  <Grid item key={-1}>
+                    <Participant participant={userStatus.user} color={COLOR_CODE.PRIMARY} />
+                  </Grid>
+                  {appointment.acceptParticipants
+                    .filter((p) => p.userId !== userStatus.user?.userId)
+                    .map((participant, index) => (
+                      <Grid item key={index}>
+                        <Participant participant={participant} />
+                      </Grid>
+                    ))}
+                </Grid>
```

Comment:
The design of this in figma is confusing, so I've just asked Oat to clarify this.
1. The user with "orange" border is "host" and other already-confirm participant is "black" border.
2. The "pending" users' images should not appear in this section yet. (I'm not sure is this one a backend problem since the pending user is appearing)

Reply:
Could you picture the participants who should be on the page pls?

Reply:
In the pic below (regardless of border pic in 1.), this appointment neither confirm nor reject by me or jah (left and right users) yet which means both of us are "pending" users but it appears in the `Participant List` which shouldn't.
Do I serve your request?
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/62459518/228556995-831522fc-7e05-4e36-a1a8-d8e680d0a8e2.png">

Reply:
yes, exactly at the point. I rechecked the code
![image](https://user-images.githubusercontent.com/69895436/228559172-59114c64-822c-475d-9472-a657f2fc9f95.png)
and found that I use acceptParticipant, so I have no idea about this issue.

Reply:
appointment id#10 doesn't have any pending users and have 3 accepted users in DB though.

Reply:
this case, your user shouldn't access this page right?

Reply:
checked it. I think everything from stored proc to services work fine. It's these lines that add current user.
![Screenshot 2023-03-29 225604](https://user-images.githubusercontent.com/18379155/228597186-626538c3-a952-4b6a-9535-4b5e9cbcf08b.png)

Reply:
Sloved, pls check